printer_mods/KenadyDwag44/sw_lightbar/README.md

```diff
@@ -0,0 +1,6 @@
+## Voron V0 Raspberry Pi 0 Mount
```

Comment:
This readme talks about a Pi Zero bracket, but the mod is for an LED holder, please update this README to describe the LED holder mod.